src/app/app.tsx

```diff
@@ -77,7 +27,7 @@ export const routes = compose(
     mount({
         '/': route({
             title: '',
-            getView: () => <WrappedApp />,
+            getView: () => <Main />,
```

Comment:
main is native html5 element, please change name to MainContent or something that relates to its objective

Reply:
Main renamed to MainContent